README.md

```diff
@@ -50,8 +50,11 @@ docker run ...
 
 ### Project setup
 
+> Hint: If using WSL make sure that you place the project files in the linux filesystem and do NOT mount them from Windows as this leads to some problems with build performance and file-watching.
+Otherwise, Node is so slow that it is hardly possible to work
 Checkout project locally. Enter the project folder and run:
```

Comment:
We should add a blank line here. As the second part should not be part of the hint.

```suggestion

Checkout project locally.
Enter the project folder and run:
```